lib/thrift/generator/enum_generator.ex

```diff
@@ -1,47 +1,38 @@
 defmodule Thrift.Generator.EnumGenerator do
 
   def generate(name, enum) do
-    macro_defs = Enum.map(enum.values, fn {key, value} ->
+    macro_defs = Enum.map(enum.values, fn {key, _value} ->
```

Comment:
I've been trying to use comprehensions a bit more for cases like this. What do you think?

```elixir
macro_defs = for {key, _value} <- enum.values do
  # ...
end
```

Reply:
Oh, actually I've been trying to avoid them when the match can fail. The `for` behavior of silently returning an empty list has bitten me a few times. It's not fun to debug. So I've come to prefer the way `Enum.map` raises an error, even though it's a little more verbose.

EEx templates are an exception. They basically rely on `for` loops and it's not as big a deal when elements are missing from an HTML page.

It's not a strong preference though. I don't mind using the other syntax if you prefer it.

Reply:
The `for` issue should only be a problem when using data from outside the libraries control, which is not the case here. Here we know that this will be a 2-tuple because we are supplying the arguments.